src/cdomain/value/cdomains/intDomain0.ml

```diff
@@ -260,6 +264,20 @@ sig
   val shift_right : Cil.ikind -> t -> t -> t * overflow_info
 end
 
+module type Bitfield_SOverflow = 
+sig 
+
+  include SOverflow 
+
+  (* used for refinements in other domains *)
+  val is_const : t -> bool
```

Comment:
Couldn't they just use `to_int`? Or are they really cases that don't then try to get the constant value itself?